### PR TITLE
Updating KinematicBody2D "is_on" functions' descriptions

### PR DESCRIPTION
**Pull request by arthurpaulino**
From Discord, `#documentation` channel:

@arthurpaulino
Hey I just had some problems with is_on_floor() from KinematicBody2D. On the docs it says:
> Returns true if the body is on the floor. Only updates when calling move_and_slide().

I had a body that was on the floor but it hadn't collided on that exact frame, so is_on_floor() was returning false and causing some unexpected behavior on my code. Fortunately @zacryol helped me figure out the real meaning of what is_on_floor() returns. I think it would be better phrased as:
> Returns true if the body has collided with the floor on the last call of move_and_slide() and false otherwise.

Do you guys think it makes sense?

@NathanLovato 
It's been a common source of confusion. The change seems good to me, some minor tweak to better fit the writing guidelines and using the docs' bbcode:
> Returns [code]true[/code] if the body collided with the floor on the last call of [member move_and_slide]. Otherwise, returns [code]false[/code].

### CODE REVIEW

**Review by NathanLovato — APPROVED** [review]

**Comment by NathanLovato** [comment]
Could you squash your commits? You need only 1 commit for maintainers to merge your PR.

**Comment by arthurpaulino** [comment]
> Could you squash your commits? You need only 1 commit for maintainers to merge your PR.

Can it be done with the merge button using the option "squash and merge"? I don't think I can squash commits after the PR is open and before it's merged.

**Comment by aaronfranke** [comment]
You can squash them and then force push. See [this](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) article for more information.

GitHub does have a "Squash and merge" button, but it destroys history, so it's better if the PR author does it.

**Comment by arthurpaulino** [comment]
> You can squash them and then force push. See [this](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) article for more information.
> 
> GitHub does have a "Squash and merge" button, but it destroys history, so it's better if the PR author does it.

Oh yeah, I got it thanks!
The squash was pushed.

File: doc/classes/KinematicBody2D.xml
Comment (madmiraal) [inline doc/classes/KinematicBody2D.xml:73]: You've changed "**a** wall" to "**the** wall". Technically, there is a left wall and a right wall.

**Review by madmiraal — CHANGES_REQUESTED** [review]
The 3D version needs to be updated the same way too.

**Comment by arthurpaulino** [comment]
@madmiraal Changes done :tada:

**Review by NathanLovato — APPROVED** [review]

**Review by madmiraal — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

For future commits, please try to use a more descriptive commit message, e.g. the same as this PR's title would have been great.

In the `git log`, you can easily spot what change is not clear about what it's doing :)
```
[akien@cauldron godot.git (master)]$ git log --pretty=oneline --no-merges
cac4fedb2a15699979e062ffa64311249695e7f9 Respect Tree.set_icon_max_width size for drawing selection box
0c9a1a1cd4c5ebfbabe5b04e81a5ddb24b3dcb06 updating descriptions
aadc8d54a47cba2ec8c5fd40875fa77ef3d89974 Bind missing constant in VisualScriptPropertyGet
84a1817a75baf9f9c5e51a9aa42549203c50a615 Fixes leaks with shader and material
d22791c271aceeade16970bffd8e8a3a146de339 doc: Sync classref with current source
1d3aec4230c1de1a00de1b2e1b7500aa82e309d7 Free a SortShader and a ParticlesCopyShader
f98acd5cfb455d7402cd7e263251a8df41746782 Ensure grid index is valid before trying to change value.
f4eef287f927e6286516484fbb0be52a2c044131 Adds Metallic to spatial light input of visual shaders
8226cee8d875af8e89cfc79fd94cb430203867b4 Fixes HTTPClient::poll crash after connection is self-assigned
351a122029fea57f431672965d623f34262d6e11 Add ability to bind an unbind arguments to Callable.
26f5bd245c535fec5bfdd51a0f939d0a51179d85 Implement GPU Particle Collisions
```

**Comment by kleonc** [comment]
@akien-mga Cherry-pick for `3.x`?

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.1.

**Pull request merged**
